usaspending_api/search/filters/postgres/defc.py

```diff
@@ -0,0 +1,17 @@
+from django.db.models import Q
+
+from usaspending_api.common.elasticsearch.search_wrappers import AwardSearch
+from usaspending_api.common.query_with_filters import QueryWithFilters
+
+
+class DefCodes:
+    underscore_name = "def_codes"
+
+    @classmethod
+    def build_def_codes_filter(cls, queryset, filter_values):
+        query = {"def_codes": filter_values}
+        filter_query = QueryWithFilters.generate_awards_elasticsearch_query(query)
+        search = AwardSearch().filter(filter_query)
+        response = search.handle_execute()
+        award_ids = [res.to_dict()["award_id"] for res in response]
+        return Q(award_id__in=list(award_ids))
```

Comment:
Has this been performance tested in sandbox? Using elasticsearch to generate a Postgres query is an interesting idea, but that IN statement on the award ids looks like a lot of trouble.

Reply:
Haven't tested directly on sandbox, hooked my local API to nonprod ES / DEV database and didn't see any problems

Reply:
I think this is always capped at a reasonable page size, isn't it?  Like 100?  So this might not get too out of hand.